core-services/prow/02_config/opendatahub-io/notebooks/_pluginconfig.yaml

```diff
@@ -0,0 +1,53 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - opendatahub-io/notebooks
+  require_self_approval: true
+external_plugins:
+  opendatahub-io/notebooks:
+  - endpoint: http://refresh
+    events:
+    - issue_comment
+    name: refresh
+  - endpoint: http://cherrypick
+    events:
+    - issue_comment
+    - pull_request
+    name: cherrypick
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
+  - endpoint: http://backport-verifier
+    events:
+    - issue_comment
+    - pull_request
+    name: backport-verifier
+  - endpoint: http://payload-testing-prow-plugin
+    events:
+    - issue_comment
+    name: payload-testing-prow-plugin
+lgtm:
+- repos:
+  - opendatahub-io/notebooks
+  review_acts_as_lgtm: true
+plugins:
+  opendatahub-io/notebooks:
+    plugins:
+    - assign
+    - approve
+    - blunderbuss
+    - help
+    - hold
+    - jira
+    - label
+    - lgtm
+    - lifecycle
+    - override
+    - retitle
+    - skip
+    - trigger
+    - verify-owners
+    - owners-label
+    - wip
```

Comment:
The updated list of plugins for ODH repositories is:
    - approve
    - assign
    - blunderbuss
    - help
    - hold
    - jira
    - label
    - lgtm
    - lifecycle
    - override
    - retitle
    - skip
    - trigger
    - transfer-issue
    - verify-owners
    - wip